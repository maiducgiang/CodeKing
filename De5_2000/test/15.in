25
40879 65710 2990 76411 95289 99579 65264 87579 69680 62205 20350 71448 18518 67574 18300 45358 73897 30889 38072 43670 87113 31740 33111 822 57633 