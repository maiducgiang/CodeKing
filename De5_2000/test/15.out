-1 32855 1495 1777 31763 33193 4079 9731 4355 20735 10175 8931 9259 33787 4575 22679 -1 1817 4759 21835 6701 7935 3679 411 19211 
-1 2 2 43 3 3 2 3 2 3 2 2 2 2 2 2 -1 17 2 2 13 2 3 2 3 